reference/customers.v3.yml

```diff
@@ -1621,15 +1623,299 @@ paths:
         name: customerId
         in: path
         required: true
+  '/customers/metafield':
```

Comment:
Here should be two path families/patterns 

```/customers/:customerId/metafields```  (GET and POST) 
Get all metafield for customer AND create metafield

```/customers/:customerId/metafields/:metafieldId``` (GET, PUT, DELETE)
Get, Update, and delete metafield by customerId + metafield

Thank you for pointing on routing config. I found that we haven't created getAllMetafieldByCustomerId. Will add it ASAP